wikimedia/wikipedia-ios
# [ALT TEXT] Publish button tweak

**Phabricator:** 

### Notes
* Disables the publish button when waiting for a response from posting the publish
* Enables it when an error occurs - here I'm verifying if `error != nil`, I'm assuming that is not possible to have a successful publish that returns a valid revID and an error 
* I added a generic error toast so the user knows what happened

### Test Steps
1. Go through alt text with a valid edit
2. Simulate an error before publishing (turning off wifi for example)
3. Tap publish - the button should disable
4. The button should be enabled when publishing returns an error
5. Try publishing normally (with wifi on); verify it works as normal
